### PR TITLE
Python 3 compatibility fixes and limiting to Django 1.x

### PR DESCRIPTION
**Pull request by LyleScott**
I have noticed that there are a few incomplete PRs involving Python 3.x compatibility. After using this project with some Django projects of mine (and thus researching every file quite a bit), these are the changes that I had to make to work under various Python 3.x versions. I have tested the best I could (including the unittests) Python 2.7 as well to make sure I didn't bump anything. These changes are pretty safe and minimal.

I also discovered that even with these Python 3.x changes, that this library does not work with Django 2.x (Python 3.x only) so I modified `setup.oy` appropriately since it seemed to be semi-related. Let me know if I should open another PR for this.

I appreciate any feedback. Cheers.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -108,7 +110,7 @@ If you want to make an RPC and continue an existing trace, you can inject the cu
         current_span = tracer.get_span(request)
         text_carrier = {}
         opentracing_tracer.inject(span, opentracing.Format.TEXT_MAP, text_carrier)
-        for k, v in text_carrier.iteritems():
+        for k, v in text_carrier.items():
```
Comment (LyleScott) [inline README.rst]: This makes for a python 2/3 compatible example. I could have gone the route of using `six` I suppose.

**Comment by ror6ax** [comment]
@tedsuo @carlosalberto - can we have this merged?

File: README.rst (hunk 1 above)
Comment (carlosalberto) [reply README.rst]: That is fine - it's only the README, so nobody gets hurt here ;)

**Comment by carlosalberto** [comment]
Hey @LyleScott 

> Let me know if I should open another PR for this.

Please do if you have enough cycles to spend ;)

**Pull request merged**